tests/unit/amo/components/TestHomeHero.js

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+
+import HomeHero, { HomeHeroBase } from 'amo/components/HomeHero';
+import {
+  dispatchClientMetadata,
+  fakeI18n,
+  shallowUntilTarget,
+} from 'tests/unit/helpers';
+import Hero from 'ui/components/Hero';
+
+describe(__filename, () => {
+  const defaultProps = {
+    i18n: fakeI18n(),
+    store: dispatchClientMetadata().store,
+  };
+
+  function render(props) {
```

Comment:
Could you please initialize the default props in the function's signature?

Also, could you please use the arrow syntax instead of the old `function`?